proto/umee/oracle/v1beta1/genesis.proto

```diff
@@ -32,12 +31,4 @@ message FeederDelegation {
 message MissCounter {
   string validator_address = 1;
   uint64 miss_counter      = 2;
-}
-
-// TobinTax defines an denom and tobin_tax pair used in
-// oracle module's genesis state
-message TobinTax {
-  string denom     = 1;
-  string tobin_tax = 2
-      [(gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec", (gogoproto.nullable) = false];
-}
+}
```

Comment:
nit: newline.

You add the following to your VSCode config:

```json
"files.insertFinalNewline": true,
"files.trimFinalNewlines": true,
"files.eol": "\n"
```

and it'll do everything for you on save 👍